addon/object/index.js

```diff
@@ -179,7 +191,17 @@ export const computed = decoratorWithParams(function(target, key, desc, params)
 
     // Use a standard ember computed since getter/setter arrity is restricted,
     // meaning ember-macro-helpers doesn't provide any benefit
-    return emberComputed(...params, { get, set: setter });
+    if (SUPPORTS_NEW_COMPUTED) {
+      return emberComputed(...params, { get, set: setter });
+    } else {
+      return emberComputed(...params, function() {
```

Comment:
could be true, I haven't used this style in a long time 😛 I'll update it to have args, no reason to choose one or the other